seed/challenges/01-front-end-development-certification/jquery.json

```diff
@@ -549,7 +549,7 @@
         "Here's how you would rewrite and italicize the text of our heading:",
```

Comment:
Ah I thought we were intentionally leaving that in to show that we italicizes through the use of emphasis.
Will update later tonight once I get back home!